addons/calendar_sms/security/sms_security.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="ir_rule_sms_template_system" model="ir.rule">
+        <field name="name">SMS Template: system administrator CRUD on calendar event templates</field>
+        <field name="model_id" ref="sms.model_sms_template"/>
+        <field name="groups" eval="[(4, ref('base.group_system'))]"/>
```

Comment:
System already has full access on sms.template model ... what is the purpose of that rule ?

Reply:
If a user from system group has also access to ``group_sale_manager`` then he will lose that access when module ``crm_sms`` is iinstalled

https://github.com/odoo/odoo/blob/d44dfff647939022b8fe8d091bc1f6562cc47d86/addons/crm_sms/security/sms_security.xml#L2-L9

Reply:
If that is the use case you were trying to fix, that's not how it should be solved. You should have a 1=1 in sms, meaning it gives all access whatever sub-accesses. Should probably be done in another PR with its own tests.